apps/adk-orchestrator/main.py

```
@@ -33,35 +111,284 @@
+            log.info(json.dumps({
+                "event": "task_executed",
+                "agent_id": best_agent.get_name(),
+                "mode": mode,
+                "task_id": task.get("id", "unknown"),
+                "status": result.get("status", "unknown"),
+                "rationale_summary": result.get("message", {}).get("rationale_summary", "")
+            }))
+            
+            return result
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)